sdk/cosmosdb/Azure.ResourceManager.CosmosDB/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2023-02-16)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+-- Added table and gremlin restorable apis
```

Comment:
Please keep the entries. Our changelog must follow the format.
For instance we could put this in the section `Features Added`
```suggestion

### Features Added

- Added table and gremlin restorable apis
```
Could we also include the api-versions of the new apis if they changed?

Reply:
Updated change log as per suggestion.